phpstan-extension.neon

```diff
@@ -11,3 +11,7 @@ services:
         class: BrandEmbassyCodingStandard\PhpStan\Rules\Method\PhpUnitTestMethodRule
         tags:
             - phpstan.rules.rule
+    -
+        class: BrandEmbassyCodingStandard\PhpStan\Rules\Mockery\DisallowMockeryShouldReceiveRule
+        tags:
+            - phpstan.rules.rule
```

Comment:
I wasn't sure about enabling it by default, but it seems like the right call since we've accepted the devsync issue. Baseline should be regenerated and that's it.